tasks/nmg_transform.py

```diff
@@ -0,0 +1,47 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+class NgmTransForm(BaseSalesforceApiTask):
+    def _run_task(self):
+        def getId(line):
+            index1 = line.find('(') + 1
+            index2 = line.find("'", index1 + 1) + 1
+            return line[index1:index2]
+
+        def createInsertLine(id, networkId, count):
+            insertLine = "INSERT INTO \"NetworkMemberGroup\" VALUES("
+            insertLine += str(count)
+            insertLine += ','
+            insertLine += id
+            insertLine += ','
+            insertLine += networkId
+            insertLine += ');'
+            return insertLine
+
+        networkId = list()
+        ids = list()
+        count = 0
+        extractFile = open('datasets/nmg-extract.sql')
+        loadFile = open('datasets/nmg-load.sql', 'a')
+
+        for line in extractFile:
+
+            # To add additional names to look for: 
+            # Copy & paste each if statement below
+            # Replace 'Customer Community - Medical Staff' with the permission set or 
+            # profile you'd like to add a network member group for
+            if 'Customer Community - Medical Staff' in line:
+                ids.append(getId(line=line))
+            if 'Customer Community - Hospital Administrator' in line:
+                ids.append(getId(line=line))
+            if 'Customer Community - Health Authority' in line:
+                ids.append(getId(line=line))
+            if '"Network" VALUES' in line:
+                networkId.append(getId(line=line))
+
+        loadFile.write('\n')
+        for id in ids:
+            count += 1
+            loadFile.write( createInsertLine(id = id, networkId = networkId[0], count = count ))
+            loadFile.write('\n')
+
+        loadFile.close()
```

Comment:
Not a blocker, but I'm wondering if this could be achieved more expeditiously just by talking to the REST API rather than doing a bulk extract/transform/load. Can't say as I've tried to load this particular object before.

Reply:
@davidmreed yeah that would have been ideal but unfortunately you cannot manipulate `Network Member Group` through Rest API. You can only use SOAP or bulk. If there is a SOAP way that is better I'm all ears!

Reply:
That is... Nuts. I had no idea there were such objects. Thanks for the enlightenment. Bulk it is!

Reply:
@davidmreed Yeah same here. I've learned so much about communities in the last week. It's a whole other world of metadata!